monitoring/godas_plotobs.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+#Command:
+#   bash godas_obsplot.sh -x exp_name -t obs_type -s start_date -e end_date
```

Comment:
The parameters in README.md and the one here are different, for eg.,
bash godas_obsplot.sh -x exp_name -t obs_type -s start_date -e end_date 
&
bash soca_plotfield.sh -x exp_name -v state_vector -s start_date -e end_date

Reply:
fixed